### PR TITLE
Bootloader Debug for CLion

### PR DESCRIPTION
**Pull request by Lucien950**
### Summary

Change up the Clion run configs.

Major TODO is to add support for the new symbol files. There does not seem to be support for GDB config settings in Clion, which means that we manually add symbol files?? Or maybe it auto detects it. I am not sure. @gtaharaedmonds 

### Changelist 

- Clion runConfigurations

### Testing Done

- [ ] Use this with real board

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: firmware/thruna/BMS/Src/App/App_Eeprom.c [hunk 1]
```diff
@@ -129,7 +129,7 @@ EEPROM_StatusTypeDef App_Eeprom_UpdateSavedSocAddress(struct Eeprom *eeprom, uin
     }
     else
     {
-        *address = *address + 1;
+        (*address)++;
```
Comment (gtaharaedmonds) [inline firmware/thruna/BMS/Src/App/App_Eeprom.c:132]: cheeky

**Review by gtaharaedmonds — APPROVED** [review]
ok !

File: firmware/thruna/BMS/Src/App/App_Eeprom.c (hunk 1 above)
Comment (Lucien950) [reply firmware/thruna/BMS/Src/App/App_Eeprom.c:132]: it wouldn't compile otherwise because 1 is not a uint16_t

Comment (Lucien950) [reply firmware/thruna/BMS/Src/App/App_Eeprom.c:132]: idk how it got past CI

Comment (gtaharaedmonds) [reply firmware/thruna/BMS/Src/App/App_Eeprom.c:132]: that's weird it might just be a difference in compiler version

**Comment by Lucien950** [comment]
fixed in 774488f08e79941a485c2b735576e5f369e1ffde on #1147

**Pull request closed without merging**